test/navigation/FakeNavigator.scala

```diff
@@ -60,3 +60,7 @@ class FakeDepartureTransportMeansNavigator(desiredRoute: Call) extends Departure
 class FakeConsignmentItemNavigator(desiredRoute: Call) extends ConsignmentItemNavigator {
   override def nextPage(page: Page, mode: Mode, userAnswers: UserAnswers): Call = desiredRoute
 }
+
+class FakeItemNavigator(desiredRoute: Call) extends AdditionalReferenceNavigator {
```

Comment:
class can be renamed to `FakeAdditionalReferenceNavigator`

Reply:
Yep, can do.. Done